lib/fhir_models/fhirpath/evaluate.rb

```diff
@@ -287,7 +287,7 @@ def self.compute(tree, data)
             tree[index] = (previous_node == :null || previous_node.empty? rescue previous_node.nil?)
             clean_index(tree, previous_index)
           when :exists
-            tree[index] = !previous_node.nil? && previous_node != :null
+            tree[index] = !previous_node.nil? && previous_node != :null && previous_node != []
```

Comment:
Do we need to check for `{}` as well? If not, this seems good to me.

Reply:
Good thought, but I have no idea if it is possible to get a `{}` here.  I think it is good enough to just scope this fix to the one we came across for now.